cpp/src/arrow/compute/kernels/aggregate_test.cc

```diff
@@ -685,5 +687,10 @@ TYPED_TEST(TestFloatingModeKernel, Floats) {
   this->AssertModeIsNull("[]");
 }
 
+TEST_F(TestInt8ModeKernelValueRange, Basics) {
```

Comment:
Are these tests useful? `int8` should already be tested in `TestIntegerModeKernel`, right?

Reply:
Good, thank you.